### PR TITLE
Added support for Ox inv and added webhook check

### PR DESCRIPTION
**Pull request by lenzh**
Moved Webhook to Config.
Added Config.Inv for inventory support (qb ox)

### CODE REVIEW

File: config.lua
Comment (MonkeyWhisper) [inline config.lua]: Move this to server sided please, so people can't dump it
And leave it as default qb, and remove your webhook

Thanks!

**Review by MonkeyWhisper — CHANGES_REQUESTED** [review]

**Comment by lenzh** [comment]
Fixed on latest push

File: server/sv_main.lua
Comment (xFutte) [inline server/sv_main.lua]:
```suggestion
if not (Config.Inv == "qb" or Config.Inv == "ox") then 
    ConfigInvInvalid()
    return;
end
```

and then extract the print to a new local function.

**Review by xFutte — APPROVED** [review]

**Comment by xFutte** [comment]
Looks good to me. I'll let @MonkeyWhisper merge since he initially requested changes.

**Comment by xFutte** [comment]
image

Should be squashed before merging.

**Pull request merged**
